jira/0.8.5/artifacthub-pkg.yml

```diff
@@ -3,8 +3,9 @@
zKImic1xa9rmubra4EEbwV8K7a/BkEwtNGx9tRI0jcdYXiyYj0JNwx45g829N13aWmqLX1xafdZrimcmejKif0tPxuv2VQ9q4HsVu/qJQ+DJ9sqvcacHsgqZIo75jM21zc6WNJ07yt7DY+jiJOML7FfavDv7zXrYSpRjrSt2bH/CPPauB7F3XsXA9q2zWPYyeN/eMG9/s3qe5RcEFzW1DBcxjNf4l7h3QSevmUHxAb+8YP8Ak9m5XY9gIIIBBFiDpBB2LgaSryoYuFSO6P5V5XOtg6Sq4eUH2/oohFL8ZsTZIyZKVpdEdJYNLo9w+U30jn1qILtUK8K8Nem7rh49ny10curSnSlqzX98PnjYluTUfCn+SPaarNVZZNfjT/Iu7TVZq83pjrPkjuaO6DzfEIiLlm8EREAREQBERAEREBH8f+LavzeTsrNi0LlTrRFgye+uUNjaOUvcAf8ArnHoWels0OaU1MyxMh/x6XzV3tYld6pnIXBeoqZPAhY37byfw1cyqq84nTyPl+o7lk8rWD9R3LJ5VlDeRqbiZZIeM4vJy9grQCz/AJIeM4vJy9grQChW5xKnkFQOWDjOTyUXZV/KgcsHGcnkouylHnCpkQ+k79njt7QWrllGk79njt7QWrlOvuMU94REWuWBZzylwlmFKoeE9jhudEw+u60YqPy2UBZWxzfJnhA/jjcQf+rmK2i/qK6i2EIwPPwdRDIdUc8TjubI0n0BanWTCFpbE3C4qqKCYG7iwNfzSM7l46wesKddZMxTe491ERa5aEREBUWXibuqNnNUOPXCB/UqpUvyqYXFThB+Ybx07RCCNRc0kvP2nFv8IUQW5TVoo15O7NE5L+K6bxX+1eiZL+K6bxX+1ei1J85l0ciVIiLBIIiIAqjx5Hw2b/j9m1W4qvyjUxbUh+yVjT0t7kjqzetdbQ0ksQ12p8U/Y5+k43op9jXuiIvauB7F21xvavUHCPTxBb+8Id8nYcrrVNYit+Hw73+zcrlXmdNdPH7feR3NGdE/H2QXlYQwFSz6ZYWOd4Qu132m2JXqouTCcoPWi2n3bDflFSVpK6PCwTizT08hkhzw4tLbF1xYkHkvs5V7qIs1Ks6ktabu+8xCEYK0VZBERQJhERAEREBxzSBrXOOpoJO4C5Xi4JxwwdUgcDVQkkXzXOzH/YfY+hephP8AwZfJv7JWVWDQNwVtOClchOVjWLZGnUQeldDCuGqamaX1E0UbR4Thc8wbrceYBZeZo1aN2hLbdp2qfId5HlO4l2ULHE18rWxhzaWEnMB757zoMrhs0aANgJ5bCIopBiXivLXziNtxCwgzSbGs8EHwzqA6diuVoor2tlpZGcFGKhMzhZ1VIXDl4JncN9Ie4czgrBXBS07I2NjjaGsjaGtaNTWtFgBzWC51pSlrNs2ErKx8v1HcsnlawfqO5ZPKvobyupuJlkh4zi8nL2CtALP+SHjOLycvYK0AoVucSp5BUDlg4zk8lF2VfyoHLBxnJ5KLspR5wqZEPpO/Z47e0Fq5ZSpO/Z47e0Fq1Tr7jFPeERFrlgUIys4DNTQl7BeWlPCC2sstaRv2e6/hCm6/CFlOzuYaurGTVNMm+OfuGQxzXNJMQXWFzG/VwoG0WsCOYEarFlIxOdRTGWJpNFM67SNUT3H/AAncg8E9GsaYWtzZJFG2LNV0dVHKxskT2vjeLtc0ggjmIXYWWsFYYqqY3pp5YidYY4hpPKW96ekKQsymYWAt7oaecxRX7KodF7ixVFvNCKtMoOUSOJjqaieH1DgWulabthGo2dqdJu1bdOhVfhbGmvqRmz1MrmHWwEMYRyFrAAeldDB1BLPI2GBhfLIbNaPWeQDWTsU40UtsiLqX2I6+abX025effyr8U9yhYusoKShgBDpC6d8rx8uUiIEjmAs0cwCgStjLWVyDVjROS/ium8V/tXomS/ium8V/tXotKfOZesiVIiLBIIiIAo9jlgY1MHcC80Rzmc/hM6R6QFIUVlKpKlNTjmiFSCnFxeTKFRWRjTicJiZqezZTpcw6GvPKD8l3oPNrVe1lJJE7MlY5jxscLdI5RzhewwuLp4mN4Z71vX7Xev4ebxGHnQdpZbnufzsPUxIb8Oh3v9m5XAqixK+PRb3+zcrdXD0308ft95HV0X0T8fZBERcc6QREQBERAEREAREQHDURB7HMOgPaW33iyp3COR2pb8WqYZANQkDo3brtzgfQroRSjNxyIuKeZnybJlhZuqBrudssf9RBX7FkxwsdcDG87pY/6SStBIrOWkY5NFQ4DyPuuHVs7Q0f5cNyTzGRwFuhvSFZ+CcFw00Yip42xxt1NbtO1xJ0uJ5TpXfRVym5ZklFLIIiKJk+SNCok5J8Kf7X7139ivhFOE3HIjKKeZVOIOINdSVrJ5+A4NrJAcyQuN3NsNBaFayIsSk5O7MpJKyCqnKBiDXVla6eDgODcyNoz5C03aLHQGlWsiRk4u6DSasyioMlWFA5pPuWwc0n312oEHwFeqIsym5ZmIxSyCIigSCIiA4KqnZIx0cjWvjeCHNcAWuadYIOsKrcZskVyX4PkDQf8mUmw5myaTbmcDvVsopRk45GGk8zN9XiJhSM2dSSnnjzZAfsErgixPwk42FFU9MZb6XWC0uis5d9hDk0UTgfJRhCUgzmOnZtuRI/oY0263BWpitijSUDCIGkyOHdyvsXu5r6mt/0iw6dKkSKEqkpZklFIgWVHFSqrxT+5uC95Mudwji3v+DtazTfvSoH+ybCn+1+9d/Yr5RZjVlFWRhwTdzwcSsFyUtFDTzEcJEHB2Zpbpe5wsbDYQi95FW9ruSsEREMhERAEREAXXqKaORubIxj28j2hw6iiK7DdNHxIVejkV/gyFrcKRhjWtHd6GgAf4Z2BWQiLoaY6WH2rjI0tHdHL7nwQREXJOgEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREB//Z
```

Comment:
is this correct?

Reply:
Wasn't sure if it would work with Artifacthub. Therefore I just changed it.